### DISCUSSION TITLE
New version 20211202- miband

### DISCUSSION THREAD

**Discussion by Jorge-1234** [Navid200/xDrip#139] [Dexcom]
Thank you for the New version. It works with Dexcom Share, but now it doesn't work with my miband watch. Thanks

**Comment by Navid200** [comment]
MiBand, what version?

**Reply by Jorge-1234** [reply]
Miband 5

**Comment by Navid200** [comment]
MiBand 5 is not supported in the official xDrip repository.
You need to ask for support from the person you got your previous working version from.

**Reply by Jorge-1234** [reply]
Can you recomend me a cheap smartwatch supported by this version, please? It is for a child

**Reply by Navid200** [reply]
You are not the only one who will need an update for their 5 to work.  He should issue a new release soon.
I have asked people with experience with watches to come to these discussions to help.
I personally have no experience.

**Reply by Jorge-1234** [reply]
Thanks. I'm afraid that when a child is used to wearing his watch it is difficult not to use it. Thank you so much

**Comment by Jorge-1234** [comment]
There is already a new version compatible with dexcom share and miband 5 y 6. Thanks

**Reply by Navid200** [reply]
I don't understand.
Why don't you use it then?

**Reply by Jorge-1234** [reply]
I am using it now, the new version is from today. I only wanted to comment it, perhaps somebody need it.
